Makefile

```diff
@@ -96,6 +96,11 @@ beats-dashboards:
 	mkdir -p build/dashboards
 	$(foreach var,$(BEATS),cp -r $(var)/_meta/kibana/ build/dashboards/$(var)  || exit 1;)
 
+# Collects all dashboards and generates dashboard folder for https://github.com/elastic/beats-dashboards/tree/master/dashboards
```

Comment:
As far as I know I think the collection is not for beats-dashboards any more, only for packaging.